packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -1018,57 +1033,60 @@ class _ViewContentState extends State<_ViewContent> {
               color: effectiveBackgroundColor,
               surfaceTintColor: effectiveSurfaceTint,
               elevation: effectiveElevation,
-              child: ClipRect(
```

Comment:
Is clipping no longer required?

Reply:
The `Material` widget above this already has `clipBehavior: Clip.antiAlias`. This makes the `ClipRect` redundant correct?